Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8693,6 +8749,28 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets the custom name visiblity of this monster. If it's false, you only see the name when you sight the mob. If it's true, you always see the custom name.",
 				},
+				SetLeashedTo =
+				{
+					Params =
+					{
+						{
+							Name = "Entity",
+							Type = "cEntity",
+						},
+					},
+					Notes = "Leash the monster to a entity.",
+				},
+				SetUnleashed =
+				{
+					Params =
+					{
+						{
+							Name = "ShouldDropPickup",
+							Type = "boolean",
+						},
+					},
+					Notes = "Unleash the monster from an entity.",
```

Comment:
I see. No problem, I can rename cEntity.AddLeashedMob / cEntity.RemoveLeashedMob to "Leash" and "Unleash" respectively. What do you think?